lib/ddtrace/contrib/action_view/events/render_partial.rb

```diff
@@ -31,6 +31,9 @@ def process(span, _event, _id, payload)
               span.set_tag(Ext::TAG_TEMPLATE_NAME, template_name)
             end
 
+            # Measure service stats
+            Contrib::Analytics.set_measured(span)
```

Comment:
do we set a unique service name on action view today?

just curious about whether we generate metrics today for view rendering and if not, do we want to add metrics for view rendering from this?

Reply:
I don't think we do, but I think we should. I think the basic components of request timing generally breaks down into:

1. Request queuing (provided by Rack instrumentation)
2. Middleware (nothing provided)
3. Processing time (provided by ActionPack instrumentation and others like Grape/GraphQL)
4. Database time (provided by ActiveRecord instrumentation and others)
5. Rendering time (nothing provided, would be provided by ActionView instrumentation)

Adding this effectively helps users see number 5.

Reply:
k

Reply:
makes sense, as long as we are adding it purposefully